cogctl/cli/decorators.py

```diff
@@ -12,22 +12,21 @@ def wrapped(*args, **kwargs):
         try:
             return f(*args, **kwargs)
 
+        except CogctlAPICredentialsException:
+            raise click.ClickException(
+                "Must set URL, user, and password to make API calls")
+
         except requests.exceptions.ConnectionError as err:
             url = err.request.url
             raise click.ClickException(
-                "Could not establish HTTP connection to %s" % url)
+                "Could not establish HTTP connection to {}. "
+                "Please check your host, user, and password "
+                "settings.".format(url))
```

Comment:
`format` is the modern Python 3 string formatting approach, compared to the older `%` one. It's more flexible and powerful, though we're admittedly not really making use of that power here.

I was just trying to keep the code "modern".